scripts/prepare_compliance_data.py

```diff
@@ -141,6 +160,8 @@ def run(self):
         os.makedirs(sequenceOntologyDir)
         shutil.copy(os.path.join(self.inputDirectory, "sequence_ontology.txt"),
                     os.path.join(sequenceOntologyDir, "sequence_ontology.txt"))
+        if self.tempdir:
```

Comment:
What does this check mean? We should really be doing a try...finally for cleanup whenever we make temporary files in any case.  The easiest way to do it is to put in a ``cleanup`` method on your class (see #880 for an example).